packages/calling/src/CallingClient/line/line.test.ts

```diff
@@ -0,0 +1,185 @@
+
+    const registrationPayload = <WebexRequestPayload>(<unknown>{
+      statusCode: 200,
+      body: mockRegistrationBody,
+    });
+
+    const mockBody = {
+      userId: webex.internal.device.userId,
+      clientDeviceUri: webex.internal.device.url,
+      serviceData: {
+        domain: '',
+        indicator: 'calling',
+      },
+    };
+
+    beforeEach(() => {
+      line = new Line(
+        userId,
+        clientDeviceUri,
+        LineStatus.ACTIVE,
+        mutex,
+        primaryMobiusUris(),
+        backupMobiusUris(),
+        LOGGER.INFO
+      );
+    });
+
+    afterEach(() => {
+      jest.clearAllTimers();
+      jest.clearAllMocks();
+      jest.useRealTimers();
+    });
+
+    it('verify successful Registration cases and keepalive', async () => {
+      jest.useFakeTimers();
+      webex.request.mockReturnValue(registrationPayload);
+
+      expect(createRegistrationSpy).toBeCalledWith(
+        webex,
+        defaultServiceData,
+        expect.any(Mutex),
+        expect.anything(),
+        LOGGER.INFO
+      );
+      expect(line.getRegistrationStatus()).toEqual(MobiusStatus.DEFAULT);
+      await line.register();
+
+      expect(webex.request).toBeCalledTimes(1);
+      expect(handleErrorSpy).not.toBeCalled();
+
+      expect(line.getRegistrationStatus()).toEqual(MobiusStatus.ACTIVE);
+      expect(line.getActiveMobiusUrl()).toEqual(primaryUrl);
+      expect(line.getLoggingLevel()).toEqual(LOGGER.INFO);
+      expect(line.getDeviceId()).toEqual(mockRegistrationBody.device.deviceId);
+
+      /** trigger sendKeepAlive and verify whether keepalive request is sent or not */
+      webex.request.mockClear();
+
+      /* specify keepalive interval as 30 seconds and advance timers by 30 */
+      line.sendKeepAlive({...mockRegistrationBody, keepaliveInterval: 30});
+
+      jest.advanceTimersByTime(30 * 1000);
+      await Promise.resolve();
+
+      expect(webex.request).toBeCalledWith({
+        ...getMockRequestTemplate(),
+        uri: `${mockRegistrationBody.device.uri}/status`,
+        method: 'POST',
+      });
+      jest.useRealTimers();
+    });
+
+    it('verify post registration api call when default mobius url is used', async () => {
+      const uri = `${webex.internal.services._serviceUrls.mobius}${URL_ENDPOINT}`;
+      webex.request.mockReturnValue(registrationPayload);
+      primaryMobiusUris.mockReturnValueOnce([uri]);
+
+      // calling it here as we need a different mobius uri to be passed in to constructor
+      line = new Line(
+        userId,
+        clientDeviceUri,
+        LineStatus.ACTIVE,
+        mutex,
+        primaryMobiusUris(),
+        backupMobiusUris(),
+        LOGGER.WARN
+      );
+
+      expect(createRegistrationSpy).toBeCalledWith(
+        webex,
+        defaultServiceData,
+        expect.any(Mutex),
+        expect.anything(),
+        LOGGER.WARN
+      );
+      expect(line.getRegistrationStatus()).toEqual(MobiusStatus.DEFAULT);
```

Comment:
Just noticed that the registration status is reflecting MobiusStatus enum, I want to discuss because we also have LineStatus created now

Reply:
We can discuss but imo registration status will be related to Mobius only.

Reply:
@Kesari3008  isn't it confusing to have both `LineStatus` and `MobiusStatus` enums in SDK since they fundamentally mean the same thing (at least thats how they are being used interchangeably)?

Reply:
LineStatus and MobiusStatus are nothing but enums created in calling SDK and reflect the same thing. This is redundant so we can remove one of the enums and use the left one. 
This change is not necessary to be done in this PR, can be taken care later

Reply:
As suggested, not making this change as part of this test case PR.

Reply:
Thanks!